include/mbedtls/ecdsa.h

```diff
@@ -235,6 +235,29 @@ int mbedtls_ecdsa_write_signature_det( mbedtls_ecdsa_context *ctx,
 #endif /* MBEDTLS_DEPRECATED_REMOVED */
 #endif /* MBEDTLS_ECDSA_DETERMINISTIC */
 
+/**
+ * \brief           Convert a signature from ASN.1 to a raw concatenation
+ *                  of {r,s}
```

Comment:
The documentation of this function should clarify the input format: it's the same format produced by `mbedtls_ecdsa_write_signature`. See https://github.com/ARMmbed/mbedtls/pull/1293#discussion_r167616489

Reply:
Done